integrationTests/cypress/integration/cas_rest.feature

```diff
@@ -0,0 +1,47 @@
+Feature: API-based CAS login and logout functionality
+
+  @requires_testuser
+  Scenario: user can access the API with username and password
+    Given the user is logged out of the CES
+    Then the user can access the SonarQube API with username and password
+
+  @requires_testuser
+  Scenario: user can not access the API with wrong username and password
+    Given the user is logged out of the CES
+    Then the user can not access the SonarQube API with wrong username and password
+
+  @requires_testuser
+  @requires_api_token_to_be_removed_afterwards
+  Scenario: API authentication with User Token
+    Given the user is member of the admin user group
+    When the user can create a User Token via the Web API
+    Then the user can access the Web API with the User Token
+
+  @requires_testuser
+  Scenario: /users API responds with correct user attributes
+    When the user's attributes are requested via Web API
```

Comment:
I tried to circumvent the "the user requests her/his attributes" discussion by switching to passive voice. I don't know if "their" is correct here, as it normally is a term for "multiple users".
I agree on using "the /users endpoint" instead of "web api" as it is more precise.